src/proposals/MultisigProposal.sol

```diff
@@ -9,31 +9,59 @@ import {Constants} from "@utils/Constants.sol";
 abstract contract MultisigProposal is Proposal {
     using Address for address;
 
-    bytes32 public constant MULTISIG_BYTECODE_HASH =
-        bytes32(0xb89c1b3bdf2cf8827818646bce9a8f6e372885f8c55e5c07acbd307cb133b000);
-
     struct Call3Value {
         address target;
         bool allowFailure;
         uint256 value;
         bytes callData;
     }
 
+    /// @notice get operations for each action, override this to provide custom operations
+    function getOperations()
+        public
+        view
+        virtual
+        returns (uint8[] memory operations)
+    {
+        uint256 actionsLength = actions.length;
+        operations = new uint8[](actionsLength);
+
+        // Default all operations to 0 (Call operation)
+        for (uint256 i = 0; i < actionsLength; i++) {
+            operations[i] = 0;
+        }
```

Comment:
free memory slots are always 0, but this is client side code so ig it doesn't matter